webapp/client/src/pages/PersonAHasDisabilityPage.test.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import PersonAHasDisabilityPage from "./PersonAHasDisabilityPage";
+import { Default as StepFormDefault } from "../stories/StepForm.stories";
+
+describe("PersonAHasDisabilityPage", () => {
+  let props;
+
+  beforeEach(() => {
+    props = {
+      stepHeader: {
+        title: "title",
+      },
+      form: {
+        ...StepFormDefault.args,
+      },
+      fields: {
+        personA_hasDisability: {
+          value: "yes",
+          errors: [],
+        },
+      },
+      prevUrl: "prevUrl",
+      numOfUsers: 1,
```

Comment:
Just because we did it differently before: If you don't have a strong opinion on this, maybe rename it to `numUsers`?

Reply:
Renamed :+1: